doc/whats_new/older_versions.rst

```diff
@@ -877,7 +879,7 @@ People
 - 127  `Jake Vanderplas`_
 - 120  `Mathieu Blondel`_
 - 85  `Alexandre Passos`_
-- 67  `Alexandre Gramfort`_
+- 68  `Alexandre Gramfort`_
```

Comment:
that's not what that counts ;) I suggest just leaving this file alone, I was just complaining.

Reply:
Can you undo this and the changelog fix please?